optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/ProblemBenchmarkResult.java

```diff
@@ -79,6 +78,9 @@
     private Long variableCount = null;
     private Long problemScale = null;
 
+    @XStreamOmitField // loaded lazily from singleBenchmarkResults
+    private Boolean hasNonDefaultSubSingleCount = null;
```

Comment:
Fixed. `maximumSubSingleCount` is the best name here, as it gives us more potential information for the future at no cost. Kept the `boolean ...()` method though, just renamed it to `isMaximumSubSingleCountMultiple()`. Moved the calculation into `accumulateResults`, specifically `determineTotalsAndAveragesAndRanking` to only take the maximum from successful benchmarks.